src/components/StreamerConnection.vue

```diff
@@ -59,6 +59,12 @@ export default {
       false,
     );
   },
+  onDestory() {
+
+    clearInterval(this.connection.looper);
```

Comment:
pretty sure `this.connection.looper` is undefined (the looper in this file is not defined on connection, but is just a local variable), buuuut....